org.mixedrealitytoolkit.uxcore/Slider/SliderSounds.cs

```diff
@@ -109,6 +109,13 @@ public class SliderSounds : MonoBehaviour
         /// </summary>
         public float MinSecondsBetweenTicks { get => minSecondsBetweenTicks; set => minSecondsBetweenTicks = value; }
 
+        [SerializeField]
+        [Tooltip("The source of the grab and release sounds")]
+        private AudioSource grabReleaseAudioSource = null;
```

Comment:
I have some questions about the original design of the sounds here.

Why they need for two audio sources?   What case would require two or more?

Reply:
Also...this code in Start() could be improved, assuming using a single AudioSource works. If both `grabReleaseAudioSource` and `grabRelease` are null, create a single audio source instead of two.

May be change 

```
            if (grabReleaseAudioSource == null)
            {
                grabReleaseAudioSource = gameObject.AddComponent<AudioSource>();
            }
            if (passNotchAudioSource == null)
            {
                passNotchAudioSource = gameObject.AddComponent<AudioSource>();
            }
```

to

```
            if (grabReleaseAudioSource == null && passNotchAudioSource == null)
            {
                grabReleaseAudioSource = passNotchAudioSource  = gameObject.AddComponent<AudioSource>();
            }
            else if (grabReleaseAudioSource == null)
            {
                grabReleaseAudioSource = passNotchAudioSource ;
            }
            else if (passNotchAudioSource == null)
            {
                passNotchAudioSource = grabReleaseAudioSource;
            }
```

Reply:
> I have some questions about the original design of the sounds here.
> 
> Why they need for two audio sources? What case would require two or more?

This is the first time i have looked at this script and I had similar questions. For this change, I strove for the minimum amount of changes.

Reply:
Please let me know if you believe making a larger scale change is needed here. If so, I will investigate / test.